Marlin/ultralcd_impl_HD44780.h

```diff
@@ -418,7 +431,11 @@ static void lcd_set_custom_characters(
           createChar_P(LCD_BEDTEMP_CHAR, bedTemp);
           createChar_P(LCD_DEGREE_CHAR, degree);
           createChar_P(LCD_STR_THERMOMETER[0], thermometer);
-          createChar_P(LCD_FEEDRATE_CHAR, feedrate);
+          #if ENABLED(LCD_SHOW_FANSPEED)
+            createChar_P(LCD_FEEDRATE_CHAR, fanspeed);
```

Comment:
Maybe put in a conditional in Conditionals_LCD.h line 321 for LCD_FANSPEED_CHAR and use that instead of overloading the LCD_FEEDRATE_CHAR?
#if ENABLED(LCD_SHOW_FANSPEED)
#define   #define LCD_FANSPEED_CHAR    0x06
#else 
  #define LCD_FEEDRATE_CHAR    0x06
#endif

Reply:
Thank you for this suggestion!
I had already had tried to use the #if construct with the 1.1.x branch, but I got the following error: "'LCD_FANSPEED_CHAR' was not declared in this scope"
Since it seems to work fine in the bugfix branch I implemented it again.